pkg/asset/manifests/scheduler.go

```diff
@@ -0,0 +1,76 @@
+package manifests
+
+import (
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	schedulerCfgFilename = filepath.Join(manifestDir, "cluster-scheduler-02-config.yml")
+)
+
+// Scheduler generates the cluster-scheduler-*.yml files.
+type Scheduler struct {
+	FileList []*asset.File
+}
+
+var _ asset.WritableAsset = (*Scheduler)(nil)
+
+// Name returns a human friendly name for the asset.
+func (*Scheduler) Name() string {
+	return "Scheduler Config"
+}
+
+// Dependencies returns all of the dependencies directly needed to generate
+// the asset.
+func (*Scheduler) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.InstallConfig{},
+	}
+}
+
+// Generate generates the scheduler config and its CRD.
+func (ing *Scheduler) Generate(dependencies asset.Parents) error {
```

Comment:
Change `ing` to something more appropriate for this asset.

Reply:
bet you can't tell where I copied and pasted from?!